src/FSharpVSPowerTools.Logic/QuickInfoMargin.xaml

```diff
@@ -0,0 +1,13 @@
+﻿<UserControl xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:sys="clr-namespace:System;assembly=mscorlib" 
+             xmlns:vs="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.11.0" 
+             xmlns:pt="clr-namespace:FSharpVSPowerTools.QuickInfo"
+             mc:Ignorable="d" 
+             Height="24"
+             d:DesignHeight="25" d:DesignWidth="600">
+    <TextBox IsEnabled="True" IsReadOnly="True" Text="{Binding QuickInfo, Mode=OneWay}" Background="{DynamicResource {x:Static SystemColors.ControlBrushKey}}" />
```

Comment:
@TeaDrivenDev I've done what they suggest there, but it now crash with this:

![image](https://cloud.githubusercontent.com/assets/873919/7784459/fe6ffac2-016e-11e5-8cc3-e16c94106486.png)

Any ideas?